doc/nrf/app_dev/build_and_config_system/index.rst

```diff
@@ -165,7 +165,7 @@ MCUboot output build files
 +------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 | File                                                             | Description                                                                                                                                                                                                                                                        |
 +==================================================================+====================================================================================================================================================================================================================================================================+
-| :file:`dfu_application.zip`                                      | Contains both the MCUboot-compatible update image for one or more cores and a manifest describing its contents (:file:`app_update.bin` and a :file:`manifest.json` file).                                                                                          |
+| :file:`dfu_application.zip`                                      | Contains both the MCUboot-compatible update image for one or more cores (and slots when MCUboot is in the Direct-XIP mode) and a manifest describing its contents (all related :file:`*.bin` files and a :file:`manifest.json` file).                              |
```

Comment:
@mkapala-nordic I think might have to stick to the version discussed offline

Contains the following:
* The MCUboot-compatible update image for one or more cores when MCUboot is not in the Direct-XIP mode and a manifest describing its contents (all related :file:`*.bin` files and a :file:`manifest.json` file).
* The MCUboot-compatible update image for the primary and secondary slots when MCUboot is in the Direct-XIP mode and manifest describing its contents (all related :file:`*.bin` files and a :file:`manifest.json` file).

Unless we could rephrase it to something like this if it doesn't affect the meaning:

Contains a manifest describing all related :file:`*.bin` files and a :file:`manifest.json` file as well as the MCUboot-compatible update image for the following:
* One or more cores when MCUboot is not in the Direct-XIP mode
* The primary and secondary slots when MCUboot is in the Direct-XIP mode

Reply:
thanks, let's stick with the first version